Tangem/Modules/Main/ActionButtonsBuy/ViewModel/ActionButtonsBuyViewModel.swift

```diff
@@ -37,12 +45,20 @@ final class ActionButtonsBuyViewModel: ObservableObject {
             ActionButtonsAnalyticsService.trackCloseButtonTap(source: .buy)
             coordinator?.dismiss()
         case .didTapToken(let token):
-            ActionButtonsAnalyticsService.trackTokenClicked(.buy, tokenSymbol: token.symbol)
             openBuy(token)
         }
     }
 
     private func openBuy(_ token: ActionButtonsTokenSelectorItem) {
+        if let disabledLocalizedReason = userWalletModel.config.getDisabledLocalizedReason(
+            for: .exchange
+        ) {
+            alert = .init(title: "", message: disabledLocalizedReason)
```

Comment:
Вообще вот так везде 
```suggestion
            alert = AlertBuilder.makeDemoAlert(disabledLocalizedReason)
```